### PR TITLE
Add memory stats for virtio-{non}-transitional memballoon

### PR DESCRIPTION
**Pull request by meinaLi**
Add memory stats for virtio-{non}-transitional memballoon

Signed-off-by: Meina Li <meili@redhat.com>

### CODE REVIEW

**Comment by meinaLi** [comment]
```
# avocado run --vt-type libvirt --vt-machine-type q35 virtio_transitional_mem_balloon
JOB ID     : 5a0bb6b388f95f52738261e04ce3c39a8dd88fc5
JOB LOG    : /root/avocado/job-results/job-2021-01-25T20.35-5a0bb6b/job.log
 (1/4) type_specific.io-github-autotest-libvirt.virtio_transitional_mem_balloon.default.virtio: PASS (64.58 s)
 (2/4) type_specific.io-github-autotest-libvirt.virtio_transitional_mem_balloon.default.virtio_transitional: PASS (76.62 s)
 (3/4) type_specific.io-github-autotest-libvirt.virtio_transitional_mem_balloon.default.virtio_non_transitional: PASS (77.04 s)
 (4/4) type_specific.io-github-autotest-libvirt.virtio_transitional_mem_balloon.rhel6_guest.virtio_transitional: PASS (69.16 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 290.84 s
```

File: libvirt/tests/src/virtio_transitional/virtio_transitional_mem_balloon.py [hunk 1]
```diff
@@ -68,6 +68,14 @@ def run(test, params, env):
             vm.start()
         is_windows_guest = (params['os_type'] == 'Windows')
         session = vm.wait_for_login()
+        # Check memory statistic
+        if libvirt_version.version_compare(6, 6, 0):
+            if (os_variant != 'rhel6' or
+                    'rhel6' not in params.get("shortname")):
+                rs = virsh.dommemstat(vm_name, ignore_status=True,
+                                      debug=True).stdout
```
Comment (chunfuwen) [inline libvirt/tests/src/virtio_transitional/virtio_transitional_mem_balloon.py:76]: change stdout to stdout_text

Comment (meinaLi) [reply libvirt/tests/src/virtio_transitional/virtio_transitional_mem_balloon.py]: Updated.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**